### PR TITLE
Experimental promo for the producer platform

### PR DESCRIPTION
**Pull request by teolemon**
Experimental promo for the producer platform

### CODE REVIEW

File: lang/fr/texts/index-pro.html [hunk 1]
```diff
@@ -35,6 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
...
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><b>Nouveau!</b>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
+
+<p>L'Éco-score arrive bientôt ! Le moteur d'Open Food Facts utilise la formule officielle de l'Éco-score.</p>
+  <ul>
+	  <li>Renseignez les données permetttant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
+    <li>Obtenez simplement et gratuitement le score sur l'ensemble de votre gamme.</li>
+   <ul>
```
Comment (hangy) [inline lang/fr/texts/index-pro.html:45]:
```suggestion
   </ul>
```

File: lang/fr/texts/index-pro.html [hunk 2]
```diff
@@ -35,6 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><b>Nouveau!</b>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
```
Comment (hangy) [inline lang/fr/texts/index-pro.html:39]: https://www.w3.org/International/questions/qa-b-and-i-tags

**Review by hangy — COMMENTED** [review]
Does this work with Crowdin? Shouldn't the French translation be overwritten by the English original?

File: lang/fr/texts/index-pro.html [hunk 3]
```diff
@@ -35,6 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
...
+
+<p>L'Éco-score arrive bientôt ! Le moteur d'Open Food Facts utilise la formule officielle de l'Éco-score.</p>
```
Comment (stephanegigandet) [inline lang/fr/texts/index-pro.html:41]: "Le moteur d'Open Food Facts utilise la formule officielle de l'Éco-score" -> it sounds very weird to talk about an official formula of something that we don't really present. What is the Eco-score?

File: lang/fr/texts/index-pro.html [hunk 4]
```diff
@@ -35,6 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
...
+  <ul>
+	  <li>Renseignez les données permetttant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
```
Comment (stephanegigandet) [inline lang/fr/texts/index-pro.html:43]:
```suggestion
	  <li>Renseignez les données permettant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
```

Same here: it's very confusing if we don't say which data is needed.

File: lang/fr/texts/index-pro.html [hunk 5]
```diff
@@ -35,8 +35,14 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><b>Nouveau!</b>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
+
+<p>L'Éco-score arrive bientôt ! Le moteur d'Open Food Facts utilise la formule officielle de l'Éco-score.</p>
+  <ul>
+	  <li>Renseignez les données permettant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
```
Comment (teolemon) [inline lang/fr/texts/index-pro.html:43]:
```suggestion
	  <li>Renseignez les données permettant d'affiner le calcul de l'Éco-score pour obtenir tous les points que méritent vos produits.</li>
	  <ul>
	  <li>Origine géographique et proportions des ingrédients</li>
	  <li>Labels environnementaux et de qualité</li>
	  </ul>
```

File: lang/fr/texts/index-pro.html (hunk 3 above)
Comment (teolemon) [reply lang/fr/texts/index-pro.html:41]: it's mostly for SEO tbh. It does not sound very natural indeed.

File: lang/fr/texts/index-pro.html (hunk 2 above)
Comment (teolemon) [reply lang/fr/texts/index-pro.html:39]: that would be <em> ?

File: lang/fr/texts/index-pro.html [hunk 6]
```diff
@@ -35,8 +35,19 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><b>Nouveau!</b>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
```
Comment (teolemon) [inline lang/fr/texts/index-pro.html:39]:
```suggestion
  <h2><em>Nouveau!</em>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
```

File: lang/fr/texts/index-pro.html [hunk 7]
```diff
@@ -35,8 +35,19 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
...
+
+<p>L'Éco-score arrive bientôt ! Le moteur d'Open Food Facts utilise la formule officielle de l'Éco-score.</p>
```
Comment (teolemon) [inline lang/fr/texts/index-pro.html:41]:
```suggestion
<p>La notation environnementale Éco-score arrive bientôt ! Vous pourrez en obtenir le calcul directement sur la plateforme avec la formule officielle de l'Éco-score.</p>
```

Comment (teolemon) [reply lang/fr/texts/index-pro.html:41]: @stephanegigandet is the wording better ?

File: lang/fr/texts/index-pro.html [hunk 8]
```diff
@@ -35,8 +35,19 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><em>Nouveau!</em>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
+
+<p>La notation environnementale Éco-score arrive bientôt ! Vous pourrez en obtenir le calcul directement sur la plateforme avec la formule officielle de l'Éco-score.</p>
```
Comment (teolemon) [inline lang/fr/texts/index-pro.html:41]:
```suggestion
<p>La notation environnementale Éco-score est arrivée ! Vous pourrez en obtenir le calcul directement sur la plateforme avec la formule officielle de l'Éco-score.</p>
```

File: lang/fr/texts/index-pro.html [hunk 9]
```diff
@@ -35,8 +35,19 @@ <h2>Trouvez des opportunités d'amélioration de la qualité des produits</h2>
 	<li>Améliorez significativement le Nutri-Score avec une réduction raisonnable du sucre, du sel ou des acides gras saturés.</li>
 	<li>Détectez les produits avec un moins bon profil nutritionnel que les produits similaires du marché.</li>
 </ul>
+  
+  <h2><em>Nouveau!</em>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score avant sa sortie</h2>
```
Comment (teolemon) [inline lang/fr/texts/index-pro.html:39]:
```suggestion
  <h2><em>Nouveau!</em>&nbsp;Complétez les données nécessaires au calcul de l'Éco-score</h2>
```

**Comment by teolemon** [comment]
@hangy I've added an English version.

File: lang/en/texts/index-pro.html [hunk 10]
```diff
@@ -35,6 +35,19 @@ <h2>Find opportunities to improve the quality of your products</h2>
 	<li>Significantly improve the Nutri-Score with a reasonable reduction in sugar, salt or saturated fatty acids.</li>
 	<li>Detect products with a lower nutritional profile than similar products on the market.</li>
 </ul>
+  
+ <h2><em>New!</em> &nbsp; Complete the data necessary to calculate the Eco-score for products sold in France.</h2>
```
Comment (stephanegigandet) [inline lang/en/texts/index-pro.html:39]: That sounds very weird. They won't see the Eco-Score unless they access fr.pro.openfoodfacts.org

I think it's best not to talk about the Eco-Score except on the French platform.

Comment (teolemon) [reply lang/en/texts/index-pro.html:39]: It was a request from Hangy. I will delete mentions to the Eco-Score

**Review by stephanegigandet — APPROVED** [review]

File: lang/en/texts/index-pro.html [hunk 11]
```diff
@@ -35,6 +35,19 @@ <h2>Find opportunities to improve the quality of your products</h2>
...
+<p>The Eco-score environmental rating has arrived! You can obtain the calculation directly on the platform with the official Eco-score formula.</p>
+   <ul>
+<li>Fill in the data to refine the Eco-score calculation to get all the points your products deserve.</li>
+<ul>
+<li>Geographical origin and proportions of ingredients</li>
+<li>Environmental and quality labels</li>
+<li>Packaging</li>
+</ul>
+     <li>Get the score easily and for free across your entire range.</li>
+    </ul>
```
Comment (teolemon) [inline lang/en/texts/index-pro.html:50]:
```suggestion
```

File: lang/en/texts/index-pro.html [hunk 12]
```diff
@@ -35,8 +35,10 @@ <h2>Find opportunities to improve the quality of your products</h2>
 	<li>Significantly improve the Nutri-Score with a reasonable reduction in sugar, salt or saturated fatty acids.</li>
 	<li>Detect products with a lower nutritional profile than similar products on the market.</li>
 </ul>
+  
+ <h2><em>New!</em> &nbsp; Complete the data necessary to calculate the Eco-score for products sold in France.</h2>
```
Comment (teolemon) [inline lang/en/texts/index-pro.html:39]:
```suggestion
</ul>
```

Comment (teolemon) [inline lang/en/texts/index-pro.html:39]:
```suggestion
</ul>
```

Comment (teolemon) [inline lang/en/texts/index-pro.html:39]:
```suggestion
</ul>
  

```

**Pull request merged**